whats_new/ose_3_0_release_notes.adoc

```diff
@@ -93,3 +93,149 @@ link:../rest_api/kubernetes_v1.html#v1-rbdvolumesource[RADOS (Ceph)].
 - https://bugzilla.redhat.com/show_bug.cgi?id=1233540[BZ#1233540]: Persistent
 Volume does not get recycled when the persistentVolumeReclaimPolicy is
 "Recycle"
+
+These are the release notes for xPaaS images on OpenShift Enterprise
+3.0. For the release notes for OpenShift Enterprise 3.0, please refer
+https://access.redhat.com/beta/documentation/en/openshift-enterprise-30-whats-new#release-notes[here].
+
+xPaaS images for OpenShift Enterprise 3.0 are provided for Red Hat JBoss
+Enterprise Application Platform, Red Hat JBoss Web Server and Red Hat
+JBoss A-MQ.
```

Comment:
I am not sure what went wrong here. I did a local build and it worked and displayed ok. :( Fixing.